ods_ci/docs/release.md

```diff
@@ -1,6 +1,6 @@
 # How To Create An ods-ci Release
 
-As an example, for RHODS release 1.21.0-21 we would create a branch releases/1.21.0-21 and add a 1.21.0 tag (without -21).
+As an example, for RHODS release 1.21-21 we would create a branch release-1.21-21 and add a 1.21 tag (without -21).
```

Comment:
Maybe it will be better to update the text with the latest releases(release-1.21 was deleted a long time ago)
@jgarciao @bdattoma what do you think?

NOTE: This idea is relevant to all changes in this PR

Reply:
Since the following MR: 
https://gitlab.cee.redhat.com/ods/jenkins/-/merge_requests/1128 merged 

To avoid failures we could fix them in the future if needed